lib/src/rules/cascade_invocations.dart

```diff
@@ -53,30 +54,50 @@ someReference
 
 ''';
 
-SimpleIdentifier _findTarget(AstNode node) {
-  Expression expression;
-  if (node is ExpressionStatement) {
-    expression = node.expression;
-  }
-
-  if (expression is MethodInvocation &&
-      expression.realTarget is SimpleIdentifier) {
-    return expression.realTarget;
+Element _getElementFromVariableDeclarationStatement(
+    VariableDeclarationStatement statement) {
+  final variables = statement.variables.variables;
+  if (variables.length == 1) {
+    return DartTypeUtilities.getCanonicalElementFromIdentifier(
+        statement.variables.variables.first.name);
```

Comment:
You've already assigned `statement.variables.variables` to a local variable, so minimally this could be cleaned up to be `variables.first.name`.

But it would be better to just replace the whole statement with `return variables.first.element`.